ramon: cwd                  /home/gh1/A/_work/z3-poly-testing/z3-poly-testing
ramon: start                Sat Oct 26 15:01:27 2024
ramon: nproc                32
ramon: sys.mem              63442 MiB
ramon: sys.mem.free         26976 MiB
ramon: sys.mem.avail        62608 MiB
ramon: sys.nprocs           3066
ramon: argv                 0 = z3
ramon: argv                 1 = inputs/QF_NIA_SAT/From_T2__dumper.t2__p19133_terminationG_0.smt2
ramon: argv                 2 = -T:20
ramon: argv                 3 = -v:2
ramon: argv                 4 = -st
ramon: argv                 5 = tactic.default_tactic=(then simplify propagate-values solve-eqs simplify smt)
ramon: argv                 6 = sat.smt=true
ramon: argv                 7 = model_validate=true
ramon: childpid             481021
(sat.copy :learned 0)
(sat.solver)
(sat.copy :learned 0)
(sat.stats   :conflicts  :restarts         :learned/bin    :gc          :time)
(sat.stats          :decisions :clauses/bin     :units         :memory       )
(sat.stats      0      0    0 19245/19177     0/0 336       0    47.95   0.01)
ASSERTION VIOLATION
File: ../src/sat/smt/arith_solver.cpp
Line: 1276
Failed to verify: s().value(c) != l_true

Z3 4.13.3.0
Please file an issue with this message and more detail about how you encountered it at https://github.com/Z3Prover/z3/issues/new
ramon: end                  Sat Oct 26 15:01:28 2024
ramon: root.execname        z3
ramon: root.utime           0.480s
ramon: root.stime           0.040s
ramon: group.total          0.534s
ramon: group.utime          0.488s
ramon: group.stime          0.046s
ramon: group.mempeak        72628KiB
ramon: group.pidpeak        2
ramon: status               exited
[31mramon: exitcode             114[0m
ramon: walltime             0.560s
ramon: loadavg              0.95
